packages/application-shell-connectors/src/utils/get-mc-api-url/get-mc-api-url.ts

```diff
@@ -39,7 +39,11 @@ export default function getMcApiUrl(
   const isInferringOfMcApiUrlOnProductionDisabled =
     environment.disableInferringOfMcApiUrlOnProduction;
 
-  if (isServedByProxy && !isInferringOfMcApiUrlOnProductionDisabled)
+  if (
+    isServedByProxy &&
+    !isInferringOfMcApiUrlOnProductionDisabled &&
+    partialWindow.origin
```

Comment:
Under test we don't have an window.origin but we still want to test things without caring about the mcApiUrl. Hence, the app-shell tests broke.